client/state/billing-transactions/individual-transactions/test/reducer.js

```diff
@@ -0,0 +1,81 @@
+/** @format */
+/**
+ * Internal dependencies
+ */
+import { requesting, error, data } from '../reducer';
+import {
+	BILLING_TRANSACTION_REQUEST,
+	BILLING_TRANSACTION_REQUEST_FAILURE,
+	BILLING_TRANSACTION_REQUEST_SUCCESS,
+} from 'state/action-types';
+
+describe( 'individualTransactions reducer', () => {
+	const transactionId = 345;
+
+	describe( 'requesting', () => {
+		test( 'true if not already requesting', () => {
+			const state = requesting( false, {
```

Comment:
We should be `deepFreeze`-ing all initial state objects in the tsts to ensure no internal mutation happens in the reducer.

Reply:
Yeah, good point. Maybe worthy only for the case we decide to change this to have a more complex structure.